### DISCUSSION TITLE
RPi Zero-W Wireless Network Screen Error

### DISCUSSION THREAD

**Discussion by Bautar** [rern/rAudio#1024] [Fixed Issues]
Hello - I just updated my Pi Zero-W (Not 2!) with the latest image.  The Network button works fine when Bluetooth is turned off, however after Bluetooth is toggled on & the machine rebooted, the following error appears when clicking the Network button:

![Capture_a](https://user-images.githubusercontent.com/99979632/190527996-ff9ca339-b14d-4d49-a2fd-f925060eda2b.PNG)

In case this picture does not translate, the top of the error message reads, "Bad control character in string literal in JSON at position 243"

When Bluetooth is toggled back to Off, the Network screen returns to normal.

**Comment by rern** [comment]
Fixed with today's update.
